Code/Framework/AzCore/AzCore/Debug/Profiler.h

```diff
@@ -5,310 +5,44 @@
-#   define AZ_PROFILE_SCOPE_IDLE_DYNAMIC(category, ...) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category); AZ_PROFILE_TIMER(AZ_INTERNAL_PROF_CAT_NAME(category))
-#endif
-
-#ifndef AZ_PROFILE_EVENT_BEGIN
-// No other profiler has defined the performance markers AZ_PROFILE_EVENT_START/END, fallback to a Driller implementation (currently empty)
-#   define AZ_PROFILE_EVENT_BEGIN(category, name) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category); (void)(name)
-#   define AZ_PROFILE_EVENT_END(category) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category)
-#endif
-
 #ifndef AZ_PROFILE_INTERVAL_START
-// No other profiler has defined the performance markers AZ_PROFILE_INTERVAL_START/END, fallback to a Driller implementation (currently empty)
-#   define AZ_INTERNAL_PROF_VERIFY_INTERVAL_ID(id) static_assert(sizeof(id) <= sizeof(AZ::u64), "Interval id must be a unique value no larger than 64-bits")
-#   define AZ_PROFILE_INTERVAL_START(category, id, ...) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category); AZ_INTERNAL_PROF_VERIFY_INTERVAL_ID(id)
-#   define AZ_PROFILE_INTERVAL_START_COLORED(category, id, color, ...) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category); (void)(color); AZ_INTERNAL_PROF_VERIFY_INTERVAL_ID(id)
-#   define AZ_PROFILE_INTERVAL_END(category, id) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category); AZ_INTERNAL_PROF_VERIFY_INTERVAL_ID(id)
-#   define AZ_PROFILE_INTERVAL_SCOPED(category, id, ...) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category); AZ_INTERNAL_PROF_VERIFY_INTERVAL_ID(id)
+#   define AZ_PROFILE_INTERVAL_START(...)
+#   define AZ_PROFILE_INTERVAL_START_COLORED(...)
+#   define AZ_PROFILE_INTERVAL_END(...)
+#   define AZ_PROFILE_INTERVAL_SCOPED(...)
 #endif
 
 #ifndef AZ_PROFILE_DATAPOINT
-// No other profiler has defined the performance markers AZ_PROFILE_DATAPOINT, fallback to a Driller implementation (currently empty)
-#define AZ_PROFILE_DATAPOINT(category, value, ...) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category); static_cast<double>(value)
-#define AZ_PROFILE_DATAPOINT_PERCENT(category, value, ...) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category); static_cast<double>(value)
-#endif
-
-#ifndef AZ_PROFILE_MEMORY_ALLOC
-// No other profiler has defined the performance markers AZ_PROFILE_MEMORY_ALLOC (and friends), fall back to a Driller implementation (currently empty)
-#   define AZ_PROFILE_MEMORY_ALLOC(category, address, size, context) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category); (void)(context)
-#   define AZ_PROFILE_MEMORY_ALLOC_EX(category, filename, lineNumber, address, size, context) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category); (void)(context)
-#   define AZ_PROFILE_MEMORY_FREE(category, address) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category)
-#   define AZ_PROFILE_MEMORY_FREE_EX(category, filename, lineNumber, address) \
-        AZ_INTERNAL_PROF_VERIFY_CAT(category)
+#   define AZ_PROFILE_DATAPOINT(...)
+#   define AZ_PROFILE_DATAPOINT_PERCENT(...)
```

Comment:
Your description didn't mention these functions. What's happening with these?

Reply:
They moved to a different header (I want to separate the interval/datapoint tracing macros also at a later time)